src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -57,79 +57,84 @@ static ValueConverter()
             Register<DateTimeOffset, DateTime>(value => value.UtcDateTime);
             Register<TimeSpan, long>(value => (long)value.TotalSeconds);
 
-            Register<int, sbyte>(value => Convert.ToSByte(value, NumberFormatInfo.InvariantInfo));
-            Register<int, byte>(value => Convert.ToByte(value, NumberFormatInfo.InvariantInfo));
-            Register<int, short>(value => Convert.ToInt16(value, NumberFormatInfo.InvariantInfo));
-            Register<int, ushort>(value => Convert.ToUInt16(value, NumberFormatInfo.InvariantInfo));
+            Register<int, sbyte>(value => { checked { return (sbyte)value; } });
```

Comment:
```suggestion
            Register<int, sbyte>(value => checked((sbyte)value));
```
I would suggest using the `checked` keyword rather than the `checked` block.

See: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/checked

Reply:
I read the same page and didn't notice it 🤦

Reply:
At least you thought of it!  Coming from a VB.Net background, in which arithmetic is checked by default, I forget that arithmetic is unchecked by default in C#.